chapter1/item2.md

```diff
@@ -2,4 +2,56 @@
 
 *This item covers customization using X resource files and tools*
 
-TODO
+---
+
+X resources provide a simple yet powerful way to customize your X session. With files such as `.Xresources`, you can easily assign colors and fonts globally or for specific applications. 
```

Comment:
This is a weak explanation and still misleads the reader to think xresources are just for colors and fonts.